docs/solution-guidance/Handle-SharePoint-Online-throttling-by-using-exponential-back-off.md

```diff
@@ -13,18 +13,18 @@ _**Applies to:** Office 365 | SharePoint Online | SharePoint Server 2013_
 > [!NOTE]
 > This page references the **Core.Throttling** sample project from the SharePoint PnP group. While this project does demonstrate the exponential backoff strategy, you should make yourself aware of the `Retry-After` HTTP header that is also returned. This was added to the SharePoint REST API after the **Core.Throttling** project was created.
 >
-> You can learn more about this HTTP header in the following article: [How to: Avoid getting throttled or blocked in SharePoint Online](../general-development/how-to-avoid-getting-throttled-or-blocked-in-sharepoint-online.md)
+> You can learn more about this HTTP header in the following article: [Avoid getting throttled or blocked in SharePoint Online](../general-development/how-to-avoid-getting-throttled-or-blocked-in-sharepoint-online.md)
 
 SharePoint Online uses throttling to prevent users from over-consuming resources. When a user runs CSOM or REST code that exceeds usage limits, SharePoint Online throttles any further request from the user for a period of time.
 
-The [Core.Throttling](https://github.com/SharePoint/PnP/tree/master/Samples/Core.Throttling) code sample in the [SharePoint Patterns and Practices](https://github.com/SharePoint/PnP) organization shows how to implement the exponential back off technique to handle throttling in SharePoint Online. When you get throttled in SharePoint Online, the exponential back off technique waits progressively longer periods of time before retrying the code that was throttled.
+The [Core.Throttling](https://github.com/pnp/PnP/tree/master/Samples/Core.Throttling) code sample in the [SharePoint Patterns and Practices](https://github.com/pnp/PnP) organization shows how to implement the exponential back off technique to handle throttling in SharePoint Online. When you get throttled in SharePoint Online, the exponential back off technique waits progressively longer periods of time before retrying the code that was throttled.

```

Comment:
Remove link... sample was removed.

Reply:
@andrewconnell Should I remove complete paragraph or just first sentence in paragraph?

Reply:
I'd rewrite the paragraph to remove the two links, but instead, reference the primary doc for this topic: https://docs.microsoft.com/en-us/sharepoint/dev/general-development/how-to-avoid-getting-throttled-or-blocked-in-sharepoint-online

Reply:
Sure, thank you @andrewconnell.